dom-core/src/main/scala/org/http4s/dom/package.scala

```diff
@@ -16,13 +16,19 @@
 
 package org.http4s
 
+import cats.effect.kernel.Async
+import cats.syntax.all._
+import fs2.Stream
+import org.scalajs.dom.experimental.ReadableStream
 import org.scalajs.dom.experimental.{Headers => DomHeaders}
 
 import scala.scalajs.js.JSConverters._
+import scala.scalajs.js.typedarray.Uint8Array
+import cats.effect.kernel.Resource
 
 package object dom {
 
-  def toDomHeaders(headers: Headers): DomHeaders =
+  private[dom] def toDomHeaders(headers: Headers): DomHeaders =
```

Comment:
I had originally meant to keep these private, hope it's okay if I sneak it in here :(

Reply:
Totally.  No compatibility is promised on 1.0 until 1.0.